remote-sr/src/main/java/de/unistuttgart/iaas/amyassist/amy/remotesr/RemoteSR.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * This source file is part of the Amy open source project.
+ * For more information see github.com/AmyAssist
+ * 
+ * Copyright (c) 2018 the Amy project authors.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information see notice.md
+ */
+
+package de.unistuttgart.iaas.amyassist.amy.remotesr;
+
+import de.unistuttgart.iaas.amyassist.amy.core.configuration.ConfigurationManager;
+import de.unistuttgart.iaas.amyassist.amy.core.di.annotation.PostConstruct;
+import de.unistuttgart.iaas.amyassist.amy.core.di.annotation.Reference;
+import de.unistuttgart.iaas.amyassist.amy.core.di.annotation.Service;
+import org.apache.commons.lang3.SystemUtils;
+import org.slf4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+/**
+ * Class to initiate the remote SR
+ * @author Benno Krauß
+ */
+@Service
+public class RemoteSR {
+
+    private static final String CONFIG_NAME = "remotesr.config";
+    private static final String CHROME_DIRECTORY_CONFIG_KEY = "chrome";
+
+    private static final String MAC_CHROME_PATH = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome" + "";
+    private static final String WINDOWS_CHROME_PATH = "/Program Files (x86)/Google/Chrome/Application/chrome.exe" + "";
+
+    @Reference
+    private Logger logger;
+
+    @Reference
+    public ConfigurationManager configurationManager;
+
+    private static final String START_SR_EVENT = "START";
+
+    private SSEClient client = null;
+
+    private RemoteSRListener listener;
+
+    @PostConstruct
+    private void init() {
+        launchChrome();
+    }
+
+    /**
+     * launch the chrome browser, open the remote-SR url immediately and prevent
+     * microphone permission dialog by using a prepared user profile
+     */
+    private void launchChrome() {
+        try {
+            URL fileURL = getClass().getClassLoader().getResource("chrome_profile");
+            if (fileURL == null) {
+                throw new IOException("Couldn't get chrome_profile resource");
```

Comment:
throw a RuntimeException and don't catch it, so the caller of `launchChrome` knows that something failed